system/jlib/jtrace.cpp

```diff
@@ -1339,6 +1346,10 @@ ISpan * CTraceManager::createServerSpan(const char * name, const IProperties * h
     return new CServerSpan(name, httpHeaders, flags);
 }
 
+ISpan * CTraceManager::createServerSpan(const char * name, const IProperties * httpHeaders, const SpanTimeStamp * spanStartTimeStamp, SpanFlags flags = SpanFlags::None) const
```

Comment:
A bit inconsistent with the other methods, and I think unnecessary, to add the default value for flags SpanFlags::None here.

Reply:
Agreed, other options were considered:
- different named method (name, httpheaders, spanStartTimeStamp, SpanFlags flags = SpanFlags::None) // Allows declared starttime, and options span flags
-  createServerSpan(name, httpHeaders, SpanFlags flags, const SpanTimeStamp * spanStartTimeStamp) //Forces users to provide spanFlags but is more consistent

Ultimately Gavin and I landed on the somewhat inconsistent , but flexible version w/ optional spanflags param. How do strong is your stance on the consistency aspect, and are you ok proceeding as-is?

Reply:
The reason I thought this might be unnecessary was because it's in the .cpp file.. and doesn't look like it's used?

I'm ok with it if there's a reason for it here.  And I'm ok with the ones in the header file although I usually try to keep them to a minimum.

Reply:
@afishbeck I think I misinterpreted your comment, the default parameter here was likely a copy/paste stowaway. Thanks for catching that.